src/main/java/com/dougnoel/sentinel/steps/TextVerificationSteps.java

```diff
@@ -81,17 +83,17 @@ public static void verifyElementTextContains(String elementName, String assertio
             String elementText = getElement(elementName).getText();
             var expectedResult = SentinelStringUtils.format(
                     "Expected the {} element to {}{} the text {}. The element contained the text: {}",
```

Comment:
This is needed for the hasTooltipText you are passing below. Otherwise you will end up with the message being incorrect.
```suggestion
                    "Expected the {} element{} to {}{} the text {}. The element contained the text: {}",
```